staging/src/k8s.io/kube-aggregator/pkg/controllers/openapi/controller.go

```diff
@@ -72,6 +78,12 @@ func NewAggregationController(downloader *Downloader, openAPIAggregationManager
 	}
 
 	c.syncHandler = c.sync
+	if services := openAPIAggregationManager.GetDelegationAPIServiceNames(); services != nil {
```

Comment:
In AddAPIService, there is an if statement that skips APIService if the service is nil. I assume those are local API services aka DelegateAPIServices, right? Can't you just remove that if statement?

Reply:
The delegation targets' APIServices/OpenAPISpec aggregation happens in [BuildAndRegisterAggregator](https://github.com/kubernetes/kubernetes/blob/cd06419973297dd1f65d1e47e9a3363bfbbcc899/staging/src/k8s.io/kube-aggregator/pkg/controllers/openapi/aggregator.go#L72-L73) which happens before `AggregationController` is created. 

I think `func (c *AggregationController) AddAPIService` only gets called on new/updated APIService which triggers apiServiceInformer, but it's not called on delegation targets.